src/collections/_documentation/platforms/cocoa/index.md

```diff
@@ -90,6 +90,35 @@ _If you crash with a debugger attached, nothing will happen._
 
 Crashes are submitted only upon re-launch of the application. To view the crash in Sentry, close your app and re-launch it.
 
+## Release Health
+
+Monitor the [health of releases]({%- link _documentation/workflow/releases/index.md -%}#release-health) by observing user adoption, usage of the application, percentage of [crashes]({%- link _documentation/workflow/releases/index.md -%}#release-health-glossary), and [session data]({%- link _documentation/workflow/releases/index.md -%}#release-health-glossary). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
+To benefit from the health data provided in Releases v2, you must use at least version 5.0.0 Beta 2 of the Cocoa SDK and enable the collection of the release health metrics by adding `“enableAutoSessionTracking": true` into options during the initialization of the SDK.
+
+```objective-c
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    // Override point for customization after application launch.
+    _ = SentrySDK(options: [
+           "dsn": “YOUR_SENTRY_DSN”,
+           "debug": true,
+           "logLevel": "verbose",
+           "enableAutoSessionTracking": true
+           "sessionTrackingIntervalMillis": 30000
+    ])
+    return true
+}
+```
```

Comment:
why do we need the full init block here? would it not be ok only `enableAutoSessionTracking` if the section is actually about release health?

Reply:
I have removed options debug and logLevel. I would keep the rest because DSN is mandatory and the interval is described under the code sample.